File: 1791-richest-customer-wealth/1791-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max_amount = 0;
        // Iterate through each array
        for(int i=0; i < accounts.length; i++)
        {
            int total = 0;
            for(int j=0; j < accounts[0].length; j++)
            {
                // Find total of each customer
                total +=  accounts[i][j];
            }
            // Find max of previous sum and current total
            max_amount = Math.max(max_amount, total);
        }
        return max_amount;
    }
}

File: 1791-richest-customer-wealth/1791-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        # Initialize new array for sum of amount for each customer
        max_amount = 0

        # Find the max amount and its index is richest customer
        # Start with assumption as first one
        for account in accounts:
            total = 0
            for amount in account:
                total = total + amount
            
            if max_amount < total:
                max_amount = total
        
        return max_amount

        